internals/core_models.py

```diff
@@ -845,12 +844,20 @@ def put(self, notify=True, **kwargs):
     if notify:
       self.__notify_feature_subscribers_of_changes(amendments, is_update)
 
-    # Invalidate ramcache for the individual feature view.
-    cache_key = '%s|%s' % (Feature.DEFAULT_CACHE_KEY, self.key.integer_id())
-    ramcache.delete(cache_key)
+    # Invalidate rediscache for the individual feature view.
+    cache_key = Feature.feature_cache_key(self.key.integer_id())
+    rediscache.delete(cache_key)
 
     return key
 
+  @classmethod
```

Comment:
Since these methods don't actually use `css`, they could be top-level functions rather than class methods.   That would make them available for both `Feature` and `FeatureEntry` and shorten the calling code because it would not need the class name.